nss-day-cohort-45/hello-world-column-counts
# added initial directory structure and flag images

## changes:
1) created directories for style and images
2) added flag images to images/ directory
3) created index.html file with boilerplate html and changed title

## testing:
1) review and approve directory structure
2) ensure index.html looks fine
3) ensure flag images appear in images/ directory
